youtube_dl/extractor/googledrive.py

```diff
@@ -53,6 +64,98 @@ def _extract_url(webpage):
+
+        subtitles = {}
+        subtitle_available_tracks = self._CAPTIONS_BY_COUNTRY_XML.findall('track')
+        for subtitle_track in subtitle_available_tracks:
+            if not subtitle_track.attrib.get('lang_code'):
+                continue
+            subtitle_lang_code = subtitle_track.attrib['lang_code']
+            subtitle_format_data = []
+            for subtitle_format in self._CAPTION_FORMATS_EXT:
+                query = {
+                    'vid': video_subtitles_id,
+                    'v': video_id,
+                    'lang': subtitle_lang_code,
+                    'fmt': subtitle_format,
+                    'name': '',
+                    'kind': '',
+                }
+                subtitle_format_data.append({
+                    'url': update_url_query('https://drive.google.com/timedtext?vid=%s&v=%s&type=track&lang=%s&name&kind&fmt=%s', query),
+                    'ext': subtitle_format,
+                })
+            subtitles[subtitle_lang_code] = subtitle_format_data
+        if not subtitles:
+            self.report_warning('video doesn\'t have subtitles')
+        return subtitles
+
+    def _get_automatic_captions(self, video_id, video_subtitles_id, hl):
+        if not video_subtitles_id or not hl:
+            return None
+        if self._CAPTIONS_BY_COUNTRY_XML is None:
+            self._set_captions_data(video_id, video_subtitles_id, hl)
+            if self._CAPTIONS_BY_COUNTRY_XML is None:
+                return None
+        self.to_screen('%s: Looking for automatic captions' % video_id)
+
+        subtitle_original_track = self._CAPTIONS_BY_COUNTRY_XML.find('track')
+        if subtitle_original_track is None:
+            return None
+        if not subtitle_original_track.attrib.get('lang_code'):
+            return None
+        subtitle_original_lang_code = subtitle_original_track.attrib['lang_code']
+
+        automatic_captions = {}
+        automatic_caption_available_targets = self._CAPTIONS_BY_COUNTRY_XML.findall('target')
+        for automatic_caption_target in automatic_caption_available_targets:
+            if not automatic_caption_target.attrib.get('lang_code'):
+                continue
+            automatic_caption_lang_code = automatic_caption_target.attrib['lang_code']
+            automatic_caption_format_data = []
+            for automatic_caption_format in self._CAPTION_FORMATS_EXT:
+                query = {
+                    'vid': video_subtitles_id,
+                    'v': video_id,
+                    'lang': subtitle_original_lang_code,
+                    'fmt': automatic_caption_format,
+                    'tlang': automatic_caption_lang_code,
+                    'name': '',
+                    'kind': '',
+                }
+                automatic_caption_format_data.append({
+                    'url': update_url_query('https://drive.google.com/timedtext?vid=%s&v=%s&type=track&lang=%s&name&kind&fmt=%s&tlang=%s', query),
```

Comment:
As per the notes from issue #13619 - this is the query string that will yield the URL for automatic captions, for a given language code `tlang` and format `fmt` .